google_compute_engine_oslogin/nss_module/nss_oslogin.cc

```diff
@@ -103,4 +104,28 @@ int _nss_oslogin_getpwnam_r(const char *name, struct passwd *result,
 // nss_getpwent_r() is intentionally left unimplemented. This functionality is
 // now covered by the nss_cache binary and nss_cache module.
 
+void _nss_oslogin_getpwent_r() {}
+void _nss_oslogin_endpwent() {}
+void _nss_oslogin_setpwent() {}
```

Comment:
When I tested this on debian-9, getpwent_r needs to return a valid NSS_STATUS. Without it, every call will complain about an "illegal status in __nss_nextAborted".

In our case it's actually fine to leave these out completely, but if BSD needs these to be explicitly defined, then getpwent_r() should return NSS_STATUS_NOTFOUND, and the (end|set)pwent() methods should return NSS_STATUS_SUCCESS.